js_modules/dagster-ui/packages/ui-core/src/assets/MaterializationTag.tsx

```diff
@@ -15,19 +15,22 @@ export const MaterializationTag = ({
   assetKey: AssetKey;
   event: {timestamp: string; runId: string};
   stepKey: string | null;
-}) => (
-  <Tag intent="success">
-    <Box flex={{gap: 4, alignItems: 'center'}}>
-      <StatusCaseDot statusCase={StatusCase.MATERIALIZED} />
-      <AssetRunLink
-        assetKey={assetKey}
-        runId={event.runId}
-        event={{timestamp: event.timestamp, stepKey}}
-      >
-        <Box style={{color: Colors.textGreen()}} flex={{gap: 4}}>
-          <Timestamp timestamp={{ms: Number(event.timestamp)}} />
-        </Box>
-      </AssetRunLink>
-    </Box>
-  </Tag>
-);
+}) => {
+  const timestamp = Number(event.timestamp) / 1000;
+  return (
+    <Tag intent="success">
+      <Box flex={{gap: 4, alignItems: 'center'}}>
+        <StatusCaseDot statusCase={StatusCase.MATERIALIZED} />
+        <AssetRunLink
+          assetKey={assetKey}
+          runId={event.runId}
+          event={{timestamp: event.timestamp, stepKey}}
+        >
+          <Box style={{color: Colors.textGreen()}} flex={{gap: 4}}>
+            <TimeFromNow unixTimestamp={timestamp} />
+          </Box>
```

Comment:
There's only one flex child so I don't think the gap is going to do anything here. Also, is the `color` definitely needed? I would have expected the `Tag` intent to apply green color to its text. (So you may be able to remove the `Box` wrapper on here altogether.)

Reply:
It's rendering the default <a /> text color, which is blue.